services/html-help-service.ts

```diff
@@ -0,0 +1,161 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+
+import util = require("util");
+import Future = require("fibers/future");
+import path = require("path");
+import options = require("options");
+import marked = require("marked");
+var TerminalRenderer = require('marked-terminal');
+var chalk = require("chalk");
+
+export class HtmlHelpService implements IHtmlHelpService {
+	private static MARKDOWN_FILE_EXTENSION = ".md";
+	private static HTML_FILE_EXTENSION = ".html";
+	private static MAN_PAGE_NAME_PLACEHOLDER = "@MAN_PAGE_NAME@";
+	private static HTML_COMMAND_HELP_PLACEHOLDER = "@HTML_COMMAND_HELP@";
+	private static RELATIVE_PATH_TO_STYLES_CSS_PLACEHOLDER = "@RELATIVE_PATH_TO_STYLES_CSS@";
+
+	private pathToManPages: string;
```

Comment:
Why do you need this local variable? You can use directly `staticConfig.HTML_PAGES_DIR`